source/features/file-finder-buffer.tsx

```diff
@@ -0,0 +1,53 @@
+import './file-finder-buffer.css';
+import React from 'dom-chef';
+import features from '../libs/features';
+import select from 'select-dom';
+
+const FILE_FINDER_INPUT_SELECTOR = '.js-tree-finder > .breadcrumb > #tree-finder-field';
+
+const pjaxStartHandler = ((event: CustomEvent): void => {
+	const destinationURL: string = event.detail?.url || '';
+	if (destinationURL.includes('/find/')) {
```

Comment:
Use destinationURL.split() and test the 3rd item specifically; otherwise this simple `includes` will return true on the `sindresorhus/find` repo